hack/defaults

```diff
@@ -23,6 +23,12 @@ APISERVER_IMAGE="${APISERVER_IMAGE:-cdi-apiserver}"
 UPLOADPROXY_IMAGE="${UPLOADPROXY_IMAGE:-cdi-uploadproxy}"
 UPLOADSERVER_IMAGE="${UPLOADSERVER_IMAGE:-cdi-uploadserver}"
 
+NETWORK_ADDONS_MULTUS_IMAGE="${NETWORK_ADDONS_MULTUS_IMAGE:-docker.io/nfvpe/multus:latest}"
```

Comment:
@myakove this should help with d/s testing

Reply:
Look good, @lukas-bednar is this ok?

Reply:
Yes I will set these params accordingly. If all images are ready I can even test it today.